src/utils/config.js

```diff
@@ -1,26 +1,31 @@
-export function getEnv(env = process.env.REACT_APP_ENV) {
-  switch (env) {
+const environment = process.env.REACT_APP_ENV ?? 'prod';
+
+export function getConfig() {
+  const commonConfig = {
```

Comment:
api link for dev and prod are different, please fix it @Megha-Dev-19

Reply:
thanks for informing